java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -1151,6 +1152,9 @@ InfScoreBound.NEGATIVE_INFINITY, new ScoreBoundary(3, false), new Limit(1, 2)),
         transaction.lcsLen("key1", "key2");
         results.add(Pair.of(LCS, buildArgs("key1", "key2", "LEN")));
 
+        transaction.publish("ch1", "msg");
```

Comment:
align transaction tests to match the reference: https://github.com/aws/glide-for-redis/commit/36d5c0161f3fdf377fee3742f6843c1c9a8632ab#diff-0e76e703bbbc3d8fba066a7e2d2dfa3d620ca525655886096cf249f92e6d5ab4

Reply:
These are unit tests and we don't have unit tests in Python.

Reply:
The tests equivalent to the reference can be found here: https://github.com/aws/glide-for-redis/pull/1662/files/6e1ca63addf31b2824ccfaced1684e692ae93e3e#diff-5474286ff166c5c919479741fe9de94e61f4dd50510ea528a3458f850b229930R278-R282 and https://github.com/aws/glide-for-redis/pull/1662/files/6e1ca63addf31b2824ccfaced1684e692ae93e3e#diff-b13c4a280f42e986a97977f987192933d9bb361607ad9ab548f3ec06dabc2cefR1165-R1169

Reply:
Does this address the concern @ikolomi ?